modules/client/examples.md

```diff
@@ -3056,9 +3089,8 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 // Create an multisig client
 const client = new MultisigClient(contextPlugin);
 const canApproveParams: CanApproveParams = {
-  pluginAddress: "0x1234567890123456789012345678901234567890",
   addressOrEns: "0x1234567890123456789012345678901234567890",
```

Comment:
Same as above, could we refactor `addressOrEns` into `approverAddressOrEns`?

Reply:
same @josemarinas